yarn.lock

```diff
@@ -2187,6 +2187,13 @@
     eslint-config-airbnb-base "14.1.0"
     eslint-plugin-vue "6.2.2"
 
+"@justeat/f-alert@0.6.1":
```

Comment:
a bit strange that it was updated. and the version is still 0.6.1, not 0.7.0

Reply:
I think it's because another package in the repo is depending on `v0.6.1` of f-alert, so as the version has been bumped, it needs this reference to keep that link